orx-image-fit/src/commonMain/kotlin/ImageFit.kt

```diff
@@ -2,120 +2,133 @@ package org.openrndr.extra.imageFit
 
 import org.openrndr.draw.ColorBuffer
 import org.openrndr.draw.Drawer
+import org.openrndr.math.Matrix44
 import org.openrndr.math.Vector2
-import org.openrndr.math.map
+import org.openrndr.math.transforms.transform
 import org.openrndr.shape.Rectangle
+import kotlin.math.max
+import kotlin.math.min
 
-
+/**
+ * Available `object-fit` methods (borrowed from CSS)
+ */
 enum class FitMethod {
-    Cover,
-    Contain
+    Cover,   // Cover target area. Crop the source image if needed.
+    Contain, // Fit image in target area. Add margins if needed.
+    Fill,    // Deform source image to match the target area.
+    None     // Maintain original image scale, crop as needed.
+    // ScaleDown // Not implemented.
```

Comment:
You can use doc-comments `/** ... */` instead of those regular comments so IntelliJ will show them when hovering over an enum variant.

Reply:
Ah nice! Will try that.